visualization/app/codeCharta/ui/ribbonBar/areaSettingsPanel/areaSettingsPanel.component.html

```diff
@@ -11,7 +11,7 @@
 	<mat-checkbox (change)="setEnableFloorLabel($event)" [checked]="enableFloorLabels$ | async"> Enable Floor Labels </mat-checkbox>
 	<cc-invert-area-option></cc-invert-area-option>
 	<cc-reset-settings-button
-		[settingsKeys]="['appSettings.dynamicMargin']"
+		[settingsKeys]="['dynamicSettings.margin']"
```

Comment:
I wonder why we don't reset "invert area" and "enable floor labels" here as well? @ce-bo what do you think about a small FR for that?

Reply:
Good catch @shaman-apprentice . That would be lovely. It is more like a bug because we forget to update this list. Would you be so kind and extend it?